app/models/general_feedback.rb

```diff
@@ -2,21 +2,30 @@ class GeneralFeedback < ApplicationRecord
   include Auditor::Model
 
   enum visit_purpose: %i[other_purpose find_teaching_job list_teaching_job]
+  enum user_participation_response: %i[interested not_interested]
```

Comment:
It is best practice to add the specific integer values for each value, see discussion here: 

https://github.com/DFE-Digital/teacher-vacancy-service/pull/907#discussion_r294832864